aten/src/ATen/native/ReduceOps.cpp

```diff
@@ -1205,8 +1206,11 @@ Tensor argmin(const Tensor& self, c10::optional<int64_t> dim, bool keepdims) {
   return at::native::argmin_out(self, dim, keepdims, result);
 }
 
-static Tensor& std_var_out(Tensor& result, const Tensor& self, IntArrayRef dim, bool unbiased, bool keepdim, bool take_sqrt) {
-  TORCH_CHECK(self.device().is_cpu() || self.is_cuda(),
+static Tensor& std_var_out(
+    const char* fname, Tensor& result, const Tensor& self,
+    c10::optional<IntArrayRef> dim, c10::optional<int64_t> correction_opt,
+    bool keepdim, bool take_sqrt) {
+  TORCH_CHECK(self.device().is_cpu() || self.device().is_cuda(),
```

Comment:
While we're here, can we tweak these error messages:

"std and var only supports tensors a CPU or CUDA device, but got: ..."

and the error message for the type check needs a tweak, too:

"std and var only support floating point and complex dtypes"